### PR TITLE
feat: replace old token list

### PR DESCRIPTION
**Pull request by NikoManiac**
closes #

### CODE REVIEW

**Comment by wenluomask** [comment]
Refer to Jira: https://mask.atlassian.net/browse/MASK-330

**Review by yanzhihong23 — APPROVED** [review]

**Comment by wenluomask** [comment]
Checked.

File: packages/mask/src/extension/popups/pages/Wallet/GasSetting/GasSetting1559.tsx [hunk 1]
```diff
@@ -241,10 +241,7 @@ export const GasSetting1559 = memo(() => {
             if (value?.computedPayload._tx.maxFeePerGas && value?.computedPayload._tx.maxPriorityFeePerGas) {
                 setValue(
                     'maxPriorityFeePerGas',
-                    fromWei(
-                        new BigNumber(value.computedPayload._tx.maxPriorityFeePerGas).toString(),
-                        'gwei',
-                    ).toString(),
+                    fromWei(new BigNumber(value.computedPayload._tx.maxPriorityFeePerGas).toFixed(), 'gwei').toString(),
```
Comment (guanbinrui) [inline packages/mask/src/extension/popups/pages/Wallet/GasSetting/GasSetting1559.tsx:244]:
```suggestion
                    fromWei(value.computedPayload._tx.maxPriorityFeePerGas, 'gwei').toFixed(),
```

File: packages/mask/src/extension/popups/pages/Wallet/GasSetting/GasSetting1559.tsx [hunk 2]
```diff
@@ -283,9 +280,9 @@ export const GasSetting1559 = memo(() => {
             if (value) {
                 const config = value.payload.params.map((param) => ({
                     ...param,
-                    gas: toHex(new BigNumber(data.gasLimit).toString()),
-                    maxPriorityFeePerGas: toHex(formatGweiToWei(data.maxPriorityFeePerGas).toString()),
-                    maxFeePerGas: toHex(formatGweiToWei(data.maxFeePerGas).toString()),
+                    gas: toHex(new BigNumber(data.gasLimit).toFixed()),
+                    maxPriorityFeePerGas: toHex(formatGweiToWei(data.maxPriorityFeePerGas).toFixed()),
+                    maxFeePerGas: toHex(formatGweiToWei(data.maxFeePerGas).toFixed()),
```
Comment (guanbinrui) [inline packages/mask/src/extension/popups/pages/Wallet/GasSetting/GasSetting1559.tsx:285]:
```suggestion
                    gas: toHex(new BigNumber(data.gasLimit)),
                    maxPriorityFeePerGas: toHex(formatGweiToWei(data.maxPriorityFeePerGas)),
                    maxFeePerGas: toHex(formatGweiToWei(data.maxFeePerGas)),
```

**Review by guanbinrui — APPROVED** [review]

**Review by yanzhihong23 — APPROVED** [review]

**Pull request merged**